explore/src/main/scala/explore/tabs/ObsTabTiles.scala

```diff
@@ -230,39 +249,36 @@ object ObsTabTiles:
         ) =>
           import ctx.given
 
-          val (obsViewPot, _, observingMode, scienceReqs, chartExposureTime) =
-            obsProperties(obsView)
+          // PREVIOUS CODE WAS: (BUT WE ALWAYS HAD "none" IN itcExposureTime
+          //   val chartExposureTime                        = obsView.toOption
+          //     .flatMap(
+          //       _.get.itcExposureTime
+          //         .map(r => ItcChartExposureTime(OverridenExposureTime.FromItc, r.time, r.count))
+          //     )
+          //     .orElse(obsView.toOption.flatMap(_.get.itc.map(_.toItcChartExposureTime)))
+          //   (obsViewPot, scienceData, observingMode, scienceReqs, chartExposureTime)
```

Comment:
Do you want to keep this commented code?

Reply:
I think yes, for the moment. This is a pending fix. WDYT @cquiroz ?